components/library/gc/Makefile

```diff
@@ -58,16 +57,6 @@ COMPONENT_POST_INSTALL_ACTION= ( cd $(@D)/libatomic_ops && \
 
 unexport SHELLOPTS
 
-COMPONENT_TEST_MASTER = $(COMPONENT_TEST_RESULTS_DIR)/results-all.master
-
-COMPONENT_TEST_TRANSFORMS += \
-	'-n ' \
-	'-e "/TOTAL/p" ' \
-	'-e "/SKIP/p" ' \
-	'-e "/PASS/p" ' \
-	'-e "/FAIL/p" ' \
-	'-e "/ERROR/p" '
-
 build: $(BUILD_32_and_64)
```

Comment:
This is no longer needed.  Just set `BUILD_BITS` to `32_and_64` and you are done.